test/Fondue.test.js

```diff
@@ -74,6 +74,13 @@ describe("The loaded font", () => {
 		);
 	});
 
+	test("returns CSS @font-face information.", async () => {
+		const fondue = await fromPath(
+			"./test/fixtures/SourceCodeVariable-Roman.ttf"
+		);
+		expect(fondue.cssString).toContain("font-weight: 200 900;");
```

Comment:
Can you extend this test to cover the whole `@font-face` declaration + add a test case for a non variable font?